spec/javascripts/components/story/story_item_spec.js

```diff
@@ -80,4 +80,34 @@ describe('<StoryItem />', () => {
       expect(children).toHaveProp('title', title);
     });
   });
+
+  describe('when the story is accepted', () => {
+    it('put .Story--accepted on childrens prop className', () => {
+      const story = storyFactory({
+        collapsed: false,
+        state: 'accepted'
+      });
+      const className = 'Story--accepted';
+
+      const wrapper = shallow(<StoryItem story={story} />);
+      const children = wrapper.find(ExpandedStory);
```

Comment:
You could move this setup to `describe` 😄 
It's better to use `describe` to setup and `it` to assert.
```js
describe('when the story is accepted', () => {
  const story = storyFactory({
    collapsed: false,
    state: 'accepted'
  });
  let children;

  beforeEach(() => {
    const wrapper = shallow(<StoryItem story={story} />);
    children = wrapper.find(ExpandedStory);
  });

  it('put .Story--accepted on childrens prop className', () => {
    expect(children).toHaveProp('className', 'Story--accepted');
  })
})
```